WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -177,4 +177,12 @@ def duration_to_s(total_seconds)
 
     [hours > 0 ? "#{hours}h " : '', minutes > 0 ? "#{minutes}m " : '', format('%.2f', seconds), 's'].join
   end
+
+  def wca_id_link(wca_id, options = {})
+    if wca_id
+      content_tag :span, class: "wca-id" do
+        link_to wca_id, "#{root_url}results/p.php?i=#{wca_id}", options
```

Comment:
No, I agree. That's what the "carries the risk" comment tries to say.

You _could_ have a test that asserts that emails don't contain relative links. That might even be a good idea anyway, but really overkill for this case.